vumi/persist/tests/test_fake_redis.py

```diff
@@ -178,6 +186,14 @@ def test_zremrangebyrank_negative_stop(self):
             [('one', 1)], 'zrange', 'set', 0, -1, withscores=True)
 
     @inlineCallbacks
+    def test_zremrangebyrank_negative_stop_empty_range(self):
+        yield self.redis.zadd('set', one=1, two=2, three=3)
+        yield self.assert_redis_op(0, 'zremrangebyrank', 'set', 0, -5)
```

Comment:
Sorry, I should've been more specific, I meant negative numbers whose absolute value is greater than the length of the set, like we are doing here for the stop param with `-5`.